VultisigApp/VultisigApp/Extensions/StringExtension.swift

```diff
@@ -135,3 +135,42 @@ extension String {
         return formattedString
     }
 }
+
+extension String {
+    init(hex: String) {
```

Comment:
doesn't this `string(data:xxx,encoding:.utf8)`

Reply:
The `init(hex:)` initializer converts a hex string directly to its ASCII representation, while `String(data: encoding: .utf8)` converts `Data` to a string using UTF-8 encoding. They differ in their approach and intended use for handling hex and byte data.

Reply:
try it , it should work , and don't want to create something if the libraries already work

Reply:
fixed, tks for giving the solution.